bin/alluxio-start.sh

```diff
@@ -52,8 +52,7 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      mount | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null
-      if [[ $? -ne 0 || -z ${ALLUXIO_RAM_FOLDER} ]]; then
+      if [[ $(mount | grep -c ${ALLUXIO_RAM_FOLDER}) != 0 ]]; then
```

Comment:
@peisun1115, it'll be very helpful if you can add such function on the next change! 

In addition, is it possible to remove `alluxio.worker.data.folder` completely from Alluxio configuration and only keep `alluxio.worker.tieredstore.level{}.dirs.path` to define the storage paths. Any drawbacks or issues on this removal?